README.md

```diff
@@ -1,24 +1,49 @@
 # Google Cloud Bigtable Client for Java
 
-Java idiomatic client for [Cloud Bigtable][cloud-bigtable].
+Java idiomatic client for [Cloud Bigtable][product-docs].
 
-[![Maven](https://img.shields.io/maven-central/v/com.google.cloud/google-cloud-bigtable.svg)](https://img.shields.io/maven-central/v/com.google.cloud/google-cloud-bigtable.svg)
-[![Codacy Badge](https://api.codacy.com/project/badge/grade/9da006ad7c3a4fe1abd142e77c003917)](https://www.codacy.com/app/mziccard/google-cloud-java)
+[![Maven][maven-version-image]][maven-version-link]
+![Stability][stability-image]
 
-- [Product Documentation][bigtable-product-docs]
-- [Client Library Documentation](https://googleapis.dev/java/google-cloud-bigtable/latest/index.html)
+- [Product Documentation][product-docs]
+- [Client Library Documentation][javadocs]
 
 ## Quickstart
 
+If you are using Maven with [BOM][libraries-bom], add this to your pom.xml file
+```xml
+<dependencyManagement>
+  <dependencies>
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>libraries-bom</artifactId>
+      <version>3.4.0</version>
```

Comment:
How will this be updated?

Reply:
The same tool that creates the common templates, fetches the latest `libraries-bom` version from Maven's metadata. This automation (currently) runs nightly.